articles/Setup-Windows-Python.md

```diff
@@ -248,9 +272,12 @@ The installation steps for CNTK on Anaconda2 are identical to
 
 Just make sure you select Python 2.7 compatible wheel files from the URL table at the top of this page.
 
-#### Installing Samples and Tutorials
+### 3. Install from Nightly Builds
+If you prefer to install or upgrade CNTK from the latest nightly build rather than an official release, we offer CNTK nightly packages. You can access the CNTK packages from the latest nightly builds [here](https://cntk.ai/nightly-windows.html).
```

Comment:
Should we teach users how to uninstall cntk pip package before installing a new one ?